src/test/csi-neonsan/README.md

```diff
@@ -0,0 +1,54 @@
+# Redis
+
+## TL;DR;
+
+```console
+helm install ks-test/csi-neonsan
+```
+
+## Installing
+
+To install the chart with the release name `csi-neonsan`:
+
+```console
+helm install ks-test/csi-neonsan --name-template csi-neonsan --namespace kube-system
+```
+
+The command deploys the csi-neonsan chart on the Kubernetes cluster in the default configuration. The configuration section lists the parameters that can be configured during installation.
+
+## Uninstalling
+
+To uninstall/delete the `csi-neonsan` deployment:
+
+```console
+helm delete csi-neonsan --namespace kube-system
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the redis chart and their default values.
+
+Parameter | Description | Default
+--- | --- | ---
+`driver.name` | Name of the CSI driver | `neonsan.csi.qingcloud.com`
```

Comment:
NeonSAN is a brand of QingStor, not QingCloud

Reply:
i will replace with "neonsan.csi.qingstor.com"

Reply:
ok